system/jlib/jlog.cpp

```diff
@@ -1836,73 +1686,34 @@ void CLogMsgManager::report(const LogMsgCategory & cat, const IException * excep
     pushMsg(new LogMsg(cat, getNextID(), unknownJob, exception->errorCode(), buff.str(), 0, port, session));
```

Comment:
>2022-05-20T13:38:38.2128830Z /Users/runner/work/HPCC-Platform/HPCC-Platform/system/jlib/jlog.cpp:1686:78: error: format string is not a string literal [-Werror,-Wformat-nonliteral]
2022-05-20T13:38:38.2228940Z     pushMsg(new LogMsg(cat, getNextID(), unknownJob, exception->errorCode(), buff.str(), 0, port, session));

looks like this is causing the compile error, but not sure why only in macos

Reply:
It was a bug - I hadn't removed the 0 parameter.  Not sure why linux didn't catch it.